hooks/notification/docs/README.ArtifactHub.md

```diff
@@ -331,6 +332,78 @@ env:
     value: "somedashboard.url/findings/{{ uid }}"
 ```
 
+#### Configuration of a Trello Notification
+A Trello notification is used to create Trello cards for each finding that matches the defined rules. This allows integrating SecureCodeBox into your development workflow. Each finding will be created as a card with the following information:
+```
+Card Name: <severity>: <name>
+Card Body: <location> <category> <description> <attributes>
+```
+
+##### Requirements: 
+- Please read the [Trello API documentation](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/) for a description on how to setup your Trello key and token. You 
+- Identify your target card list where the new cards will be created and get the list ID. To find the card list and label ids use the Trello JSON hack as described here https://stackoverflow.com/questions/26552278/trello-api-getting-boards-lists-cards-information
+
+##### Configuration
+To configure a Trello notification set the `type` to `trello` and the `endPoint` to point to your env containing the Trello cards API endpoint. You also need to define the following env vars (if an env var is not defined it will take the default value, if required and not set the notification won't be sent):
+- TRELLO_CARDS_ENDPOINT: The Trello cards API endpoint. Default: https://api.trello.com/1/cards.
+- TRELLO_KEY: Your Trello API key. Reade the requirements above. Required.
+- TRELLO_TOKEN: Your Trello API token. Reade the requirements above. Required.
+- TRELLO_LIST: Id if the list where the cards will be placed. Required.
+- TRELLO_LABELS: Comma separated list of label IDs to apply to the card. If empty no labels will be applied. Default: "")
+- TRELLO_POS: The position in the list where the new card will be placed as defined by the Trello cards API. Default: top.
+
+
+##### Example Config
+
+The below example shows how to create a helm values chart and load secrets for access.
+You must have `endPoint` point to a [defined environment variable](https://github.com/secureCodeBox/secureCodeBox/blob/main/hooks/notification/hook/hook.ts#L20), not a string.
+
+```
+# cat myvalues.yaml
```

Comment:
```suggestion
# cat trello_values.yaml
```

Reply:
Done and named the other file trello_secrets.yaml